core/src/test/java/io/github/belgif/rest/guide/validator/core/ApiFunctionsTest.java

```diff
@@ -57,16 +58,16 @@ void isLowerCamelCase() {
     @Test
     void isCompatibleMediaType() {
```

Comment:
This duplicates the tests in MediaTypeTest too much. It's sufficient to test all variant values of MediaType.includes in MediaTypeTest, and only test the behavior when having a list of media types here.